### ISSUE TITLE
关于预训练的提问

### ISSUE THREAD

**Issue by hexxxl** [xhanxu/Mamba3D#3]
请问为什么你们的模型在未通过预训练的情况下在PB_T50_RS上能够达到91.81和92.64的成绩，却在经过MAE和BERT两种预训练策略之后却只能达到88.97和87.58了呢

**Comment by xhanxu** [comment]
Thank you for your interest in our work. And, that's a great question!

 As we discussed in the Discussion section, the Masked Point Modeling-based pre-training methods are not suitable for Mamba3D. We believe that Mamba3D is still a sequential model. When trained from scratch, the model can learn meaningful sequential information from the complete point cloud, resulting in a more complete shape representation. However, if the shape is masked, it can only learn incomplete (and different) information from an incomplete point cloud, preventing it from obtaining a complete shape representation. This loss of information leads to a reduction in the benefits of pre-training and may even harm the model's performance. 

In fact, we have tried supervised pre-training on the ModelNet40 dataset and then fine-tuning on the PB_T50_RS dataset. We found that the results could reach or exceed those of training from scratch, which also demonstrates the importance of global information for Mamba3D, and the huge difference between Transformer and Mamba3D. 

We will continue to research and address this issue. Does this answer your question?

**Comment by hexxxl** [comment]
Thank you for your answer. You answered my question in great detail .


I have another question. I am very interested in the voting mechanism in your point cloud model. I also saw it in the PCM article. However, none of the articles explained this vote mechanism in detail. And maybe because of my insufficient search ability, I have never been able to find where it comes from. Can you tell me which article it comes from.







------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "xhanxu/Mamba3D"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年5月21日(星期二) 中午1:59
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [xhanxu/Mamba3D] 关于预训练的提问 (Issue #3)





 
Thank you for your interest in our work. And, that's a great question!
 
As we discussed in the Discussion section, the Masked Point Modeling-based pre-training methods are not suitable for Mamba3D. We believe that Mamba3D is still a sequential model. When trained from scratch, the model can learn meaningful sequential information from the complete point cloud, resulting in a more complete shape representation. However, if the model is masked, it can only learn incomplete (and different) information from an incomplete point cloud, preventing it from obtaining a complete shape representation. This loss of information leads to a reduction in the benefits of pre-training and may even harm the model's performance.
 
In fact, we have tried supervised pre-training on the ModelNet40 dataset and then fine-tuning on the PB_T50_RS dataset. We found that the results could reach or exceed those of training from scratch, which also demonstrates the importance of global information for Mamba3D, and the huge difference between Transformer and Mamba3D.
 
We will continue to research and address this issue. Does this answer your question?

**Comment by xhanxu** [comment]
My apologies for not explaining in detail in the paper. This voting strategy is used in multiple models such as PointBERT, PointMAE, ReCon, PointMamba, etc. I believe it would be easier for you to understand by directly looking at their code.

https://github.com/Pang-Yatian/Point-MAE/blob/11f4c99f60e2f635e40d23ccaf21ec9aa42705ef/tools/runner_finetune.py#L390
https://github.com/qizekun/ReCon/blob/f22ddcbea6e6949d32cf0f98c9d1e72ceb7c845a/tools/runner_finetune.py#L380
https://github.com/RunpeiDong/ACT/blob/640d33f8fa0365d0625e79a7023a6fa7bf8529e3/tools/runner_finetune.py#L425
https://github.com/LMD0311/PointMamba/blob/c44a3da7c70417f0880b9541a787e44919d2e84f/tools/runner_finetune.py#L411

**Comment by hexxxl** [comment]
Thank you for your anwser.It's helpful for me.




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Han ***@***.***&gt;; 
发送时间: 2024年5月21日(星期二) 下午2:42
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [xhanxu/Mamba3D] 关于预训练的提问 (Issue #3)














 
My apologies for not explaining in detail in the paper. This voting strategy is used in multiple models such as PointBERT, PointMAE, ReCon, PointMamba, etc. I believe it would be easier for you to understand by directly looking at their code.
我很抱歉没有在论文中详细解释。这种投票策略用于多种模型，例如 PointBERT、PointMAE、ReCon、PointManba 等。我相信通过直接查看他们的代码，你会更容易理解。
 
https://github.com/Pang-Yatian/Point-MAE/blob/11f4c99f60e2f635e40d23ccaf21ec9aa42705ef/tools/runner_finetune.py#L390
 https://github.com/qizekun/ReCon/blob/f22ddcbea6e6949d32cf0f98c9d1e72ceb7c845a/tools/runner_finetune.py#L380
 https://github.com/RunpeiDong/ACT/blob/640d33f8fa0365d0625e79a7023a6fa7bf8529e3/tools/runner_finetune.py#L425
 https://github.com/LMD0311/PointMamba/blob/c44a3da7c70417f0880b9541a787e44919d2e84f/tools/runner_finetune.py#L411
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
直接回复此电子邮件，在 GitHub 上查看或取消订阅。
You are receiving this because you authored the thread.
您收到此消息是因为您创作了线程。  Message ID: ***@***.***&gt;
